src/controllers/AccessRequestController.ts

```diff
@@ -0,0 +1,71 @@
+import {
+    Post,
+    Route,
+    Controller,
+    TsoaResponse,
+    Res,
+    Body,
+    SuccessResponse,
+} from 'tsoa';
+import {
+    serverErrorType,
+    GenericTypeORMErrorType,
+    requiredFieldErrorType,
+    accessRequestResponseBody,
+} from '../helpers/types';
+
+import logger from '../middleware/logger';
+import { createRequest } from '../db/AccessRequest.db';
+import { TypeORMError } from 'typeorm';
+
+@Route('user-requests')
+export class AccessRequestController extends Controller {
+    @SuccessResponse('201', 'Created')
```

Comment:
This is where you would put a comment explaining the endpoint for the api docs (if you want it to be explained, that is)